rancher/distros-test-framework
# fix lint + pre commit

#### Proposed Changes ####





#### Types of Changes ####



https://github.com/rancher/distros-test-framework/pull/53#discussion_r1425158452 
 Check this comment to explanations 


- adding pre commit
- fix lint

#### Testing ####



Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.


2. Verify code lint; we should not have errors.
 - ok 

3. Update the documentation if needed.
- ok 

4. Update makefile and docker run if adding new tests.
 -ok 

5. Run your tests at least 4 times with all configurations needed and possible.
- ok

6. If needed test with different os types.

 -ok 
#### Linked Issues ####



#### Further Comments ####
